### ISSUE TITLE
[Bug] Multitool shadow

### ISSUE THREAD

**Issue by ProPeach** [tsunamayo/Starship-EVO#1997]
### Short Description of the issue:
The multitool floats in midair according to the shadow of the player model, despite being firmly in the characters hand in 3rd person mode
https://i.imgur.com/vtGelRg.mp4

### Steps to reproduce the issue:
Hold a multitool and observe the players shadow

### Environment
- Starship EVO version: 20w19d
- Graphics Card: RX 580

**Comment by tsunamayo** [comment]
Yes I know it looks terrible, it is a bit tricky to fix though ... =( Thanks

**Comment by ProPeach** [comment]
It's not too big an issue for now I don't think, no problem if it has to wait till later!
